app/lib/test_applications.rb

```diff
@@ -57,8 +57,11 @@ def create_application(states:, courses_to_apply_to: nil, apply_again: false, co
     end
 
     Audited.audit_class.as_user(candidate) do
+      traits = %i[with_safeguarding_issues_disclosed with_no_safeguarding_issues_to_declare].sample
+
       @application_form = FactoryBot.create(
         :completed_application_form,
+        *traits,
```

Comment:
Unlike below, won't this just be a symbol anyway? could be passed in directly without a splat